docker-compose-all.yml

```diff
@@ -96,13 +96,12 @@ services:
       - SPRING_PROFILES_ACTIVE=noqueue
 
   localstack:
-    # pinned to 10.4 due to https://github.com/localstack/localstack/issues/1992
-    image: localstack/localstack:0.10.4
+    image: localstack/localstack:0.12.9.1
     networks:
       - hmpps
     container_name: localstack
     ports:
-      - "4567-4597:4567-4597"
+      - "4566-4597:4566-4597"
```

Comment:
Why the change of ports?

Reply:
Rather than a port for each service they have consolidated all services on the same port.